templates/gulp/gulp_helper.rb

```diff
@@ -0,0 +1,15 @@
+module GulpHelper
+
+  def gulp_asset_path(path)
+    path = REV_MANIFEST[path] if defined?(REV_MANIFEST)
+    "/assets/#{path}"
+  end
+
+  def image_tag(image, options = {})
+    source = "images/#{image}"
+    alt = File.basename(image, File.extname(image)).capitalize
```

Comment:
@geoRG77 Are your sure, that alt is not working by default? Because it looks like the default behaviour remove digits from suffix of file.
http://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_alt